pkg/storaged/crypto-keyslots.jsx

```diff
@@ -21,9 +21,10 @@ import cockpit from "cockpit";
 import React from "react";
 
 import {
-    Card, CardBody, CardTitle, CardHeader, CardActions,
+    Button, Card, CardBody, CardTitle, CardHeader, CardActions, Checkbox,
+    Form, FormGroup,
     DataListItem, DataListItemRow, DataListItemCells, DataListCell, DataList,
-    Text, TextVariants
+    Text, TextVariants, Modal, TextInput as TextInputPF, Stack,
```

Comment:
Why you renamed the import?

Reply:
Because TextInput is imported also from `storagelib.js` a few lines below